README.md

```
@@ -84,8 +84,44 @@
 
 ### FROM
 
-- Currently, only `FROM FRONTMATTER_DATA("<path>")` is supported.
-- This will find all markdown files in the specified path and use their frontmatter data as rows.
+#### FRONTMATTER_DATA
+
+- `FROM FRONTMATTER_DATA("<path>")`
+- This will find all markdown files in the specified `<path>` and use their frontmatter data as rows.
+- FIELDS:
+    - `file.name` - name of the file
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)